scripts/setup_sgtm_tasks_project.py

```diff
@@ -20,11 +20,14 @@ class CustomField:
     enum_options: List[EnumOption]
 
 
+# If adding a new option, ensure that the color you're selecting is part of the list of colors in this [file](https://github.com/Asana/codez/blob/next-master/asana2/asana/web/ui/colors/color_helpers.ts)
```

Comment:
💯

Reply:
Actually, is there a public-facing list anywhere in the API documentation? Non-Asana people won't have access to this reference. If not, maybe inline the options that are available today (even though it'll get out of date)

Reply:
From what I can see, it looks like we don't have the list anywhere in the docs.

The color field for [EnumOption](https://developers.asana.com/docs/enum-option) is just typed as a `string`. Maybe this is an opportunity to improve our docs. I'll raise it in #eng-random or something like that.

For now, I'll just include the inline list.